game.py

```diff
@@ -529,6 +530,16 @@ def main():
     shoot_counter = 0
 
     while running:
+        ship_data = n.send(p1_server_data_resp)
+        try:
+            modules = ship_data['modules']
+            # Modules are using capitals for    
+            engine_power = ship_data['Engines']
+            o2_power = ship_data['O2']
+
```

Comment:
oops, forgot to do the rest. Missing radar and weapon element. Otherwise i think this looks good